experimenter/experimenter/nimbus_ui/templates/nimbus_experiments/home.html

```diff
@@ -103,7 +104,21 @@ <h5 class="fw-semibold">
                       </td>
                       <td>{{ experiment.get_application_display }}</td>
                       <td>{{ experiment.home_type_choice }}</td>
-                      <td>{{ experiment.get_channel_display }}</td>
+                      <td>
+                        {% render_channel_icons experiment as channel_data %}
+                        {% if channel_data %}
+                          <div class="d-flex flex-wrap">
+                            {% for channel in channel_data %}
+                              <span class="d-inline-flex align-items-center{% if channel.is_multi %} me-2 mb-1{% endif %}">
+                                <span class="{{ channel.icon_info.color }} me-1">
+                                  <i class="{{ channel.icon_info.icon }}"></i>
+                                </span>
+                                <span class="small">{{ channel.label }}</span>
+                              </span>
+                            {% endfor %}
```

Comment:
Yeah exactly, when the data is in the right shape, it's very easy to loop over and spit out into HTML.  Keeps the html all in one place etc.  Very nice 🙏 🙏 🙏